client/src/app/containers/Game/index.tsx

```diff
@@ -0,0 +1,175 @@
+
+import { useAuth } from '../../../auth/useAuth';
+import ActionBox from './ActionBox';
+import CribHand from './CribHand';
+import { Game, Phase } from './models';
+import PlayerHand from './PlayerHand';
+import PlayingCard from './PlayingCard';
+import ScoreBoard from './ScoreBoard';
+import { useGame } from './useGame';
+
+const showCutCard = (phase: Phase) =>
+    !['Deal', 'BuildCrib', 'Cut'].includes(phase);
+
+const handsExist = (phase: Phase) =>
+    !['unknownPhase', 'Deal', 'DealingReady'].includes(phase);
+
+const handForPlayer = (
+    game: Game,
+    myID: string,
+    position: 'across' | 'right' | 'left',
+) => {
+    if (!handsExist(game.phase)) {
+        return [];
+    }
+    const numPlayers = game.teams.reduce(
+        (prev, team) => prev + team.players.length,
+        0,
+    );
+    const isFourPlayer = numPlayers === 4;
+    if (position === 'across') {
+        if (game.teams.length === 3) {
+            const secondPlayerID = game.teams.filter(
+                t => !t.players.some(p => p.id === myID),
+            )[1].players[0].id;
+            return game.hands[secondPlayerID] ?? [];
+        }
+        if (isFourPlayer) {
+            const partnerID = game.teams
+                .filter(t => t.players.some(p => p.id === myID))[0]
+                .players.filter(p => p.id !== myID)[0].id;
+            return game.hands[partnerID] ?? [];
+        }
+        const opponentID = game.teams.filter(
+            t => !t.players.some(p => p.id === myID),
+        )[0].players[0].id;
+        return game.hands[opponentID] ?? [];
+    }
+    if (position === 'right') {
+        if (isFourPlayer) {
+            const rightID = game.teams
+                .filter(t => t.players.some(p => p.id !== myID))[0]
+                .players.filter(p => p.id !== myID)[1].id;
+            return game.hands[rightID] ?? [];
+        }
+        // nothing!
+        return [];
+    }
+    if (position !== 'left' || !isFourPlayer) {
+        return [];
+    }
+    // position is left
+    const leftID = game.teams
+        .filter(t => t.players.some(p => p.id !== myID))[0]
+        .players.filter(p => p.id !== myID)[0].id;
+    return game.hands[leftID] ?? [];
+};
+
+const GamePage: React.FunctionComponent = () => {
+    const { game, refreshGame } = useGame();
+    const { currentUser } = useAuth();
+    const myHand = handsExist(game.phase) ? game.hands[currentUser.id] : [];
+
+    return (
+        <Grid container xl spacing={1} direction='row' justify='space-between'>
+            <Grid
+                item
+                container
+                md
+                spacing={2}
+                direction='column'
+                align-content='space-between'
+            >
+                <Grid item xs sm container>
+                    <PlayerHand
+                        phase={game.phase}
+                        hand={handForPlayer(game, currentUser.id, 'across')}
+                        pegged={game.pegged_cards}
```

Comment:
:mag: for `PlayerHand`, we should just use `useGame` and access the phase and pegged cards. Additionally, we could pass in `playerID` instead of "hand" and allow each component to grab the cards for the given player, right? maybe this is a follow-up...